recipe/meta.yaml

```diff
@@ -24,20 +24,20 @@ requirements:
   run:
     - python >=3.5
     - pyzmq >=17.1.2
-    - typeguard
+    - typeguard >=2.5
     - ipyparallel
     - globus-sdk
     - dill
     - tblib
     - ipykernel
     - requests
     - paramiko
-    - psutil
-    - sqlalchemy ==1.3.3
+    - psutil >=5.5.1
+    - sqlalchemy >=1.3.0,!=1.3.4
     - sqlalchemy-utils
     - pydot
     - networkx
-    - Flask
+    - flask >=1.0.2
```

Comment:
@ZhuozhaoLi Can you check if this is right? I know we've had issues with `flask` before.

Reply:
Yes, this is right.